### PR TITLE
Decode GB2312 (Simplified Chinese)

### PR DESCRIPTION
**Pull request by xs9627**
Some Chinese feed encoding with gb2312, like: http://tech.qq.com/web/rss_web.xml

The Chinese character will be garbled when http.get with UTF-8. 
So here the code fix get the chunk data without encoding, then decode with [iconv-lite](https://github.com/ashtuchkin/iconv-lite).

### CODE REVIEW

**Comment by bobby-brennan** [comment]
Hi there,

First off - very much appreciate the pull request, test case etc. But unfortunately, we've decided against including iconv-lite for now (per [discussion here](https://github.com/bobby-brennan/rss-parser/pull/57)), as it more than doubles the size of the browser distribution (~190K to ~450K).

However, I'd be open to finding a way to create two distributions, one with iconv-lite and one without. I will think more about this, and will leave this PR open for now.

-Bobby

File: test/parser.js [hunk 1]
```diff
@@ -266,4 +266,34 @@ describe('Parser', function() {
       })
     })
   })
+
+  it('should respect timeout option', function(done) {
```
Comment (rbren) [inline test/parser.js:270]: Looks like you duplicated some tests here?

**Comment by rbren** [comment]
Sorry, I was looking at the wrong commit. Disregard.

My earlier comments stand. Though I'd still be very interested in finding a way to have a second distro for non-utf8 encodings.

**Comment by xs9627** [comment]
OK, It's just to merge the latest changes to my forked repo. I used 
```"rss-parser": "https://github.com/xs9627/rss-parser/tarball/master``` as the dependence in my other project.
BTW, thanks for the awesome tool. It works all good for about 2 years :)

**Comment by rbren** [comment]
Closing in favor of https://github.com/rbren/rss-parser/issues/187

**Pull request closed without merging**